pages/_document.tsx

```diff
@@ -3,12 +3,12 @@ import Document, {
   Head,
   Main,
   NextScript,
-  NextDocumentContext
+  DocumentContext
 } from 'next/document'
 import '../theme.scss'
 
 export default class MyDocument extends Document {
-  static async getInitialProps (ctx: NextDocumentContext) {
+  static async getInitialProps (ctx: DocumentContext) {
```

Comment:
You are adding changes for next as well in future I would suggest you split out the PRs. In this case you can make two PRs one for next and one for messanger that branches out from next branch